content/docs/pipelines/pipelines-tutorial.md

```diff
@@ -0,0 +1,616 @@
+
+1. (Optional) Use `kubectl` to connect to your cluster:
+
+  * Connect your local `kubectl` session to the cluster:
+
+      ```
+      gcloud container clusters get-credentials \
+          ${DEPLOYMENT_NAME} --zone ${ZONE} --project ${PROJECT}
+      ```
+
+  * Switch to the `kubeflow` namespace to see the resources on the Kubeflow 
+    cluster:
+
+      ```
+      kubectl config set-context $(kubectl config current-context) --namespace=kubeflow
+      ```
+
+  * Check the resources deployed in the `kubeflow` namespace:
+
+      ```
+      kubectl get all
+      ```
+
+1. Access the Kubeflow UI, which becomes available at the following URI after 
+   several minutes:
+
+    ```
+    https://<deployment-name>.endpoints.<project>.cloud.goog/
+    ```
+
+    The following screenshot shows the Kubeflow UI:
+    <img src="/docs/images/central-ui.png" 
+        alt="Prediction UI"
+        class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click **Pipeline Dashboard** to access the pipelines UI. The pipelines UI
+   looks like this:
+    <img src="/docs/images/pipelines-ui.png" 
+      alt="Pipelines UI"
+      class="mt-3 mb-3 border border-info rounded">
+
+Notes:
+
+* When the deployment has finished, you should have a running cluster in the 
+  cloud ready to run your code. You can interact with the cluster either by 
+  using [`kubectl`][kubectl] or by going to the
+  [GKE page on the GCP Console][gcp-console-kubernetes-engine].
+
+* While the deployment is running, you can watch your resources appear on the
+  GCP console:
+    * [Deployment on Deployment Manager][gcp-console-deployment-manager]
+    * [Cluster on GKE][gcp-console-kubernetes-engine]
+    * [Computing resources on Compute Engine][gcp-console-compute-engine]
+
+* It can take 10-15 minutes for the URI to become available. Kubeflow needs
+  to provision a signed SSL certificate and register a DNS name.
+    * If you own/manage the domain or a subdomain with [Cloud DNS][dns]
+      then you can configure this process to be much faster.
+    * While you wait you can access Kubeflow services by using `kubectl proxy`
+      and `kubectl port-forward` to connect to services in the cluster.
+
+### Create a Cloud Storage bucket
+
+The next step is to create a Cloud Storage bucket to hold your trained model
+
+[Cloud Storage][cloud-storage] is a scalable, fully-managed object/blob store.
+You can use it for a range of scenarios including serving website content,
+storing data for archival and disaster recovery, or distributing large data
+objects to users via direct download. This tutorial uses Cloud Storage to
+hold the trained machine learning model and associated data.
+
+Use the [`gsutil mb`][gsutil-mb] command to create a storage bucket. Your
+*bucket name* must be unique across all of Cloud Storage.
+The following commands create a bucket in the `us-central1` region,
+which corresponds to the `us-central1-c` zone used earlier
```

Comment:
Since you're using zone as an env var, it's probably better to grab the region programatically:
`REGION=$(gcloud compute zones describe $ZONE  --format="value(region.basename())")`

Reply:
Nice. Done.